src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,33 @@
+// prop-types
+import Proptypes from "prop-types";
+
+// css
+import "./button.css";
+
+export const TYPES = {
+  PRIMARY: "primary",
+  SECONDARY: "secondary",
+};
+
+const Button = (props) => {
+  const { type = TYPES.PRIMARY, label, className, ...extra } = props;
+  return (
+    <button {...extra} className={`btn ${type} ${className}`}>
```

Comment:
Suggestion: There should be mapping of label vs class here e.g.
`const buttonClassNames = cx({
    [styles.primaryButton]: view === BUTTON_VIEWS.PRIMARY,
    [styles.secondaryButton]: view === BUTTON_VIEWS.SECONDARY,
    [styles.tertiaryButton]: view === BUTTON_VIEWS.TERTIARY,
    [styles.iconButton]: view === BUTTON_VIEWS.ICON,
    [styles.linkButton]: view === BUTTON_VIEWS.LINK,
    [styles.tertiaryButtonHover]: view === BUTTON_VIEWS.TERTIARY && highlightOnHover,
  });`